packages/ui/src/kit/VInput/VInput.vue

```diff
@@ -19,12 +22,27 @@
       v-bind="$attrs"
       v-on="$listeners"
     >
+      <v-progress-circle
```

Comment:
using `progress-circle` only for one loader icon is overhead.
maybe use the same story as it was defined in `button-loader-molecule`?

Reply:
and kit components can't use other components

Reply:
for now, this is only component that show animated loader
then I suggest to turn it into molecule, and `VProgressCircle` will just wrapper over molecule